packages/dredd/test/fixtures/json-schema-draft-7-boolean.apib

```diff
@@ -0,0 +1,21 @@
+FORMAT: 1A
+
+# Machines API
+
+# Group Machines
+
+# Machines collection [/machines/{id}]
+    + Parameters
+        + id (number, `1`)
+
+## Get Machines [GET]
+
++ Request (application/json)
+    + Parameters
+        + id (number, `2`)
```

Comment:
```suggestion
        + id: 2 (number)
```

Reply:
Got it. Copy pasted from another blueprint, perhaps it was different intentionally. Will fix.

Reply:
Fixed in https://github.com/apiaryio/dredd/pull/1622/commits/6534cf61609145345da38a0674c220a37d825204.